packages/components/src/ui/create-styles/create-style-system/constants.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Uses the the prefix for the CSS variables compiled by the system.
+ */
+export const NAMESPACE = '--wp-g2';
+
+export const DARK_MODE_ATTR_PROP = 'data-system-ui-mode';
+export const HIGH_CONTRAST_MODE_ATTR_PROP = 'data-system-ui-contrast-mode';
+export const COLOR_BLIND_MODE_ATTR_PROP = 'data-system-ui-color-blind-mode';
+export const REDUCED_MOTION_MODE_ATTR_PROP =
+	'data-system-ui-reduced-motion-mode';
+
+export const DARK_MODE_ATTR = `[${ DARK_MODE_ATTR_PROP }="dark"]`;
+export const HIGH_CONTRAST_MODE_MODE_ATTR = `[${ HIGH_CONTRAST_MODE_ATTR_PROP }="high"]`;
+
+export const COLOR_BLIND_MODE_ATTR = `[${ COLOR_BLIND_MODE_ATTR_PROP }="true"]`;
+export const REDUCED_MOTION_MODE_ATTR = `[${ REDUCED_MOTION_MODE_ATTR_PROP }="true"]`;
+
+export const DARK_HIGH_CONTRAST_MODE_MODE_ATTR = `${ DARK_MODE_ATTR }${ HIGH_CONTRAST_MODE_MODE_ATTR }`;
+
+export const MODE_SPECIFICITY_COMPOUND_LEVEL = 3;
+
+export const INTERPOLATION_CLASS_NAME = '__interpolationClassName__';
```

Comment:
Seems like some of the variables here are not useful anymore now that we dropped the ThemeProvider right?

Reply:
I think you're right! I'll make sure I remove the unused ones.

Reply:
Actually never mind, these are all used by the global style system which is the basis of the theme variables for the new style system (as in, the foundation for how those variables get transformed from an object passed to `createStyleSystem` and then turned into actually applied CSS custom properties)

Reply:
Some of the variables here refer explicitly to the "data" attributes that were used previously in the theme provider, do we still need these?